### PR TITLE
[develop][518221](Task) [S] Webhooks: create python script

### PR DESCRIPTION
**Pull request by arvidfm**
This PR adds an initial version of a CLI utility that allows you to manage webhooks, along with the library backing the utility and an example script showing how to use the library.

### CODE REVIEW

File: flixpy/flix/cli/main.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    )
+
+
+@click.group()
+@click.option("-s", "--server", type=str, help="The URL of the Flix server.")
+@click.option("-u", "--username", type=str, help="The username to authenticate with.")
+@click.option("-p", "--password", type=str, help="The password to authenticate with.")
+@click.pass_context
+async def flix_cli(
+    ctx: click.Context, server: str | None, username: str | None, password: str | None
```
Comment (bcvery1) [inline flixpy/flix/cli/main.py:58]: Why not `Optional[str]`?

File: flixpy/flix/lib/client.py [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+import dateutil.parser
+
+from . import signing, errors, forms
+
+
+__all__ = ["Client", "AccessKey"]
+
+
+@dataclasses.dataclass
+class AccessKey:
```
Comment (bcvery1) [inline flixpy/flix/lib/client.py:16]: Not for this ticket, but I wonder if we could generate these from openapi schemas

File: flixpy/flix/cli/main.py (hunk 1 above)
Comment (arvidfm) [reply flixpy/flix/cli/main.py:58]: It's shorter and equivalent. I did a quick search to see what was preferred, and the consensus seemed to be to prefer the union syntax if your version of Python supports it

File: flixpy/flix/lib/client.py (hunk 2 above)
Comment (arvidfm) [reply flixpy/flix/lib/client.py:16]: Good question, will look into it

**Review by bcvery1 — COMMENTED** [review]
Will the PING function be added at a later stage?

**Comment by arvidfm** [comment]
I'm doing it as part of the next task where I add an example server

**Review by bcvery1 — APPROVED** [review]

**Pull request merged**